### DISCUSSION TITLE
Is there a list of supported abbreviation for books?

### DISCUSSION THREAD

**Discussion by moyicat** [tim-hub/obsidian-bible-reference#46] [General]
For example, `rev` and `deu` works, but `isa` doesn't work for Isaiah, it's currently routing to 1 Samuels.

Is there a supported list of abbreviations? Thanks 🙏

**Comment by tim-hub** [comment]
Thanks, there are, at current stage, it is based the back-end api provider, which does not guranteer all abbreviations included, 

:( 

Will think about it, maybe there is someway to solve it

**Comment by aknisly** [comment]
The `isa` is read as a lower case Roman numeral. It took me a while to figure this out. See #79

**Comment by lukszar** [comment]
Is it possible to provide some dictionary of abbreviations in other languages?

**Reply by tim-hub** [reply]
I don't think so, but it is something we need to resolve this issue.
